src/main/java/com/iota/iri/conf/NetworkConfig.java

```diff
@@ -36,13 +36,6 @@
      */
     boolean isAutoTetheringEnabled();
 
-    /**
-     * Default Value: {@value BaseIotaConfig.Defaults#P_REMOVE_REQUEST}
-     *
-     * @return {@value NetworkConfig.Descriptions#P_REMOVE_REQUEST}
```

Comment:
Just noticed the Description still exists if you scroll down
Please remove it

Reply:
Changed